frontend/src/components/Cards/EventCard.tsx

```diff
@@ -0,0 +1,198 @@
+// EventCard.tsx:
+import React, { useState } from "react";
+import Carousel from "../eventsPage/Carousel";
+import { Launch, PhotoLibrary } from "@material-ui/icons";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+
+import {
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  CardMedia,
+  IconButton,
+  Zoom,
+} from "@material-ui/core";
+import theme from "src/theme/theme";
+
+interface EventCardProps {
+  title: string;
+  subtitle?: string;
+  thumb: string;
+  web?: string;
+  date?: string;
+  images?: Array<string>;
+  description?: string;
+  priority: boolean;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    linkContainer: {
+      "&:hover": {
+        background: "#03d7e03b",
+        color: "cyan",
+      },
+      marginTop: ".8rem",
+      width: "5rem",
+      paddingInline: ".6rem",
+      background: "#afdbdd3b",
```

Comment:
Avoid hard-coding values whenever possible. You can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).